core/assets/snapshot.go

```diff
@@ -55,7 +54,6 @@ func (s *Service) Stopped() bool {
 
 func (s *Service) serialiseActive() ([]byte, error) {
 	enabled := s.GetEnabledAssets()
-	sort.SliceStable(enabled, func(i, j int) bool { return enabled[i].ID < enabled[j].ID })
```

Comment:
Why are those not needed?

Reply:
I've moved the sorting into the `GetEnabledAssets/getPendingAssets/getPendingAssetUpdates` calls, because thats where the conversion from `map -> slice` happens.

Reply:
right